### PR TITLE
updates to figure plotting, etc.

### PR DESCRIPTION
**Pull request by jamiemkass**
I updated the code to reflect the current figures and added a maintainer for CoordinateCleaner.

### CODE REVIEW

**Comment by sylvainschmitt** [comment]
There are issues with your pull request (for instance you did not updated the functions doc: https://github.com/sylvainschmitt/sdmverse/actions/runs/8613560486/job/23605101649?pr=36#step:6:95). I'll fix them locally this time.

File: inst/extdata/packages/CoordinateCleaner.yaml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 name : CoordinateCleaner
 title: Automated Cleaning of Occurrence Records from Biological Collections
 version: 2.0-20
-author: "Alexander Zizka, Daniele Silvestro, Tobias Andermann, Josue Azevedo, Camila Duarte Ritter, Daniel Edler, Harith Farooq, Andrei Herdean, Maria Ariza, Ruud Scharn, Sten Svanteson, Niklas Wengstrom, Vera Zizka, Alexandre Antonelli, Irene Steves, Francisco Rodriguez-Sanchez"
+author: Alexander Zizka, Daniele Silvestro, Tobias Andermann, Josue Azevedo, Camila Duarte Ritter, Daniel Edler, Harith Farooq, Andrei Herdean, Maria Ariza, Ruud Scharn, Sten Svanteson, Niklas Wengstrom, Vera Zizka, Alexandre Antonelli, Irene Steves, Francisco Rodriguez-Sanchez
```
Comment (sylvainschmitt) [inline inst/extdata/packages/CoordinateCleaner.yaml:4]: @jamiemkass , this is wrong according to CRAN

Comment (jamiemkass) [reply inst/extdata/packages/CoordinateCleaner.yaml:4]: Sorry, I just removed the quotes. I guess we can put them back.

**Comment by jamiemkass** [comment]
> There are issues with your pull request (for instance you did not updated the functions doc: https://github.com/sylvainschmitt/sdmverse/actions/runs/8613560486/job/23605101649?pr=36#step:6:95). I'll fix them locally this time.

Oops, sorry. I can add these and push if you don't get around to it. Thanks.

**Pull request merged**